packages/react-components/react-tabster/src/hooks/useOnKeyboardNavigationChange.ts

```diff
@@ -11,8 +10,7 @@ import { useEventCallback } from '@fluentui/react-utilities';
  * @param callback - called every time the keyboard navigation state changes
  */
 export function useOnKeyboardNavigationChange(callback: (isNavigatingWithKeyboard: boolean) => void) {
-  const { targetDocument } = useFluent();
-  const keyborg = React.useMemo(() => targetDocument && createKeyborg(targetDocument.defaultView!), [targetDocument]);
+  const keyborg = useKeyborg();
   const eventCallback = useEventCallback(callback);
```

Comment:
```suggestion
```

Wouldn't this be the same as not using `useEventCallback` at all and then just removing `eventCallback` from the dependencies on the `useEffect`?!